README.md

```diff
@@ -15,6 +15,11 @@ This is the simplest configuration for developers to start with.
 3. Access the browseable API, starting at http://localhost:8000/api/v1/
 4. When finished, use `Ctrl+C`
 
+### Run Populate Script
+1. Get API Token from https://styx.neurology.emory.edu/girder/
+2. Run script:
+`docker-compose run -e DJANGO_API_TOKEN=[your token] --rm django ./manage.py populate`
+
```

Comment:
It's worth noting now that in the future we will likely need to change the way we pass in api keys to the import scripts. If we want to allow the user to create datasets down the road, it will need to be something we accept through the endpoint. This is fine for now, of course.

Reply:
I made an issue for this so we don't lose track of this.